packages/dds/test-dds-utils/package.json

```diff
@@ -17,10 +17,10 @@
 	"scripts": {
 		"api": "fluid-build . --task api",
 		"api-extractor:commonjs": "api-extractor run --local",
-		"api-extractor:esnext": "copyfiles -u 1 \"dist/**/*-@(alpha|beta|public|untrimmed).d.ts\" lib",
```

Comment:
We don't need the esnext build here?

Reply:
This package is not building ESM. There's no build:esnext script.

Reply:
No, I know. I guess I'm wondering why this package only wants a commonjs build. Probably not a concern for this PR, just wondering if this is something we need to track for later?